new/LOC7393Amyatsma.xml

```diff
@@ -0,0 +1,58 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7393Amyatsma" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾAmʸaṣmā</title>
```

Comment:
check the symbol - fidel has mʷ

Reply:
I wanted to encode mʸā. I took the fidal character ፙ from wikipedia -> Gəʿəz script. The character looks different in the oxygen and in Github. In wikipedia and as well in github, there is a dash on top, as it is visible in the manuscript. In Oxygen, however it is formed in a hybrid way with a dash on top as well as a dash below as if it is a combination o mʷā ሟ and mʸā ፙ.

<img width="340" alt="Screenshot 2023_Amyatsma" src="https://github.com/BetaMasaheft/Places/assets/40291787/225d146a-aa77-491f-bba7-90fcc6769c86">

Reply:
OK, thank you!